### DISCUSSION TITLE
tutorials/graphos-shipping-supergraph/01-graphos-for-schema-delivery

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#117] [course feedback]
# tutorials/graphos-shipping-supergraph/01-graphos-for-schema-delivery

https://www.apollographql.com/tutorials/graphos-shipping-supergraph/01-graphos-for-schema-delivery

**Comment by JTMonaghan** [comment]
The link to the GraphOS Basics course takes me to a "Page not found" page.

**Reply by mabuyo** [reply]
Oh yes, good catch, will update this shortly! For now, here's the correct link: https://www.apollographql.com/tutorials/getting-started-with-graphos/03-whats-in-a-supergraph
